aiida/cmdline/commands/node.py

```diff
@@ -232,79 +231,69 @@ def cat(self, *args):
             else:
                 raise
 
-
-class _Show(VerdiCommand):
-    """
-    Show node information (pk, uuid, class, inputs and outputs)
-    """
+class _Show(VerdiCommandWithSubcommands):
 
     def run(self, *args):
-        """
-        Show node information.
-        """
-        import argparse
-        from aiida.common.exceptions import NotExistent
-
-        parser = argparse.ArgumentParser(
-            prog=self.get_full_command_name(),
-            description='Show information of a node.')
-        parser.add_argument('pk', type=int, default=None, nargs="+",
-                            help="ID of the node.")
-        parser.add_argument('--print-groups', action='store_true',
-                            dest='print_groups', default=False,
-                            help="Show groups containing the nodes.")
-        parser.add_argument('--no-print-groups', '--dont-print-groups',
-                            action='store_false', dest='print_groups',
-                            default=False,
-                            help="Do not show groups containing the nodes"
-                                 "output. Default behaviour.")
-
-        args = list(args)
-        parsed_args = parser.parse_args(args)
-
-        if not is_dbenv_loaded():
-            load_dbenv()
-
-        for pk in parsed_args.pk:
-            try:
-                n = load_node(pk)
-                self.print_node_info(n, print_groups=parsed_args.print_groups)
-            except NotExistent as e:
-                print >> sys.stderr, e.message
-                sys.exit(1)
+        verdi()
+
+@node.command('show')
+@click.argument('identifiers', type=str, nargs=-1)
+@click.option('--uuid', 'uuid', flag_value=True,
```

Comment:
Would a `-u` value be accepted here as well?

Reply:
No, just -uuid. I will also add -u